internal/cmdexport/pathutil/pathutil.go

```diff
@@ -55,14 +55,14 @@ func Exists(path string) bool {
 func IsInsideDir(path string, directory string) (isInside bool, err error) {
 	if !filepath.IsAbs(path) {
 		err = fmt.Errorf(
-			"argument `path` (%s) is not an absolute path",
+			"argument `path` ('%s') is not an absolute path",
```

Comment:
what about https://www.geeksforgeeks.org/how-to-print-string-with-double-quotes-in-golang/

Reply:
Responding to the title of https://github.com/GoogleContainerTools/kpt/issues/964
Is there a strong preference for single vs. double quotes? @phanimarupaka

Reply:
@mikebz comment makes more sense in this context. We already have some examples in the codebase. https://github.com/kubernetes-sigs/kustomize/blob/master/cmd/config/internal/commands/cmdset.go#L131

Reply:
Done.